index.bs

```diff
@@ -651,3 +779,81 @@ The [=remote end steps=] are:
 
 2. Return [=success=] with data |body|
 
+### Subscribe ### {#command-session-subscribe}
+
+The <dfn export for=commands>subscribe</dfn> command enables certain events either
+globally or for a set of browsing contexts
+
+Issue: This needs to be generalised to work with realms too
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      ```
+      SubscribeCommand = {
+        method: "session.subscribe",
+        params: SubscribeParameters
+      }
+
+      SubscribeParameters = {
+        events: [text],
+        contexts?: [BrowsingContext],
+        * text => any,
```

Comment:
I'm not familiar with CDDL, what does `* text => any` mean?

Reply:
`*` means zero or more. `text => any` means a key value pair where the key is a text string and the value is any. So `* text => any` at the end of a map means the map can contain other arbitrary keys in addition to the ones that are required.

Reply:
I think we can just get rid of this instance of it.

Reply:
That means we won't ignore unknown parameters here. I thought that having this everywhere was the plan.

Reply:
Good call.